### DISCUSSION TITLE
How to style this particular border?

### DISCUSSION THREAD

**Discussion by dimitrisnl** [zed-industries/zed#12725] [Feature Requests]
Hey, I just noticed after upgrading that there's this new focus border.

![CleanShot 2024-06-06 at 12 27 51@2x](https://github.com/zed-industries/zed/assets/4951004/57ed3098-6984-4873-b3d4-4b34dbd9fb9d)

I don't recall having this before, and I couldn't find anything relevant in the changelog.

Does someone know the key I can use in the theme overrides to align it to my theme?

**Comment by failable** [comment]
Same question. Really ugly.

<img width="293" alt="image" src="https://github.com/zed-industries/zed/assets/5475262/5c505d2c-737a-4a63-8fc1-c20e7808ec32">

**Comment by th0jensen** [comment]
You can configure it under "text.accent", it's also the color used to indicate selected icons. I've also noticed it showed up not too long ago... possibly v0.137.0

**Reply by dimitrisnl** [reply]
This is pretty bad though, I need the border to be darker, but the text-accent is usually lighter to contrast the dark background :/

**Reply by failable** [reply]
I think the issue is `text.accent` is used in different UI elements (text and border). I'd like to have the text darker but the border lighter.

![image](https://github.com/zed-industries/zed/assets/5475262/5c599435-7ac0-4ed5-90ce-edbbddaac601)
![image](https://github.com/zed-industries/zed/assets/5475262/4509e424-ea99-442e-9c1d-668e07c21ae5)

**Comment by Moshyfawn** [comment]
This is an intended a11y feature, but I've passed on the feedback to possibly disable it via a setting.

Be sure to +👍 and follow #12723 to keep the conversation in one place.

**Reply by danmartinvela** [reply]
any update on this subject?

**Reply by Tom-Boscher** [reply]
Looking for update too
